queries/python/locals.scm

```diff
@@ -113,6 +111,8 @@
 (named_expression
   (identifier) @definition.var)
 
+(as_pattern 
+  alias: (identifier) @definition.var)
```

Comment:
Looks duplicated from line 37

Reply:
yes, but I removed the duplicate before merging.